src/entities/transactions/transaction.ts

```diff
@@ -86,9 +125,17 @@ export interface RainbowTransaction {
   explorerUrl?: string;
 }
 
+export enum FlashbotsStatus {
+  PENDING = 'PENDING',
+  INCLUDED = 'INCLUDED',
+  FAILED = 'FAILED',
+  CANCELLED = 'CANCELLED',
+  UNKNOWN = 'UNKNOWN',
+}
+
 export type MinedTransaction = RainbowTransaction & {
-  status: 'confirmed' | 'failed';
-  flashbotsStatus?: 'CANCELLED' | 'FAILED' | 'INCLUDED';
+  status: TransactionStatus.confirmed | TransactionStatus.failed;
+  flashbotsStatus?: FlashbotsStatus;
```

Comment:
I think `flashbotStatus` should be in `BaseTransaction`, because unless the status is `'INCLUDED'` it's never a "mined" transaction